projects/components/src/download-json/download-json.component.ts

```diff
@@ -0,0 +1,79 @@
+import { DOCUMENT } from '@angular/common';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, Inject, Input, Renderer2 } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { IconSize } from '@hypertrace/components';
+import { Observable } from 'rxjs';
+import { catchError, finalize, take } from 'rxjs/operators';
+import { ButtonSize, ButtonStyle } from '../button/button';
+import { NotificationService } from '../notification/notification.service';
+
+@Component({
+  selector: 'ht-download-json',
```

Comment:
I would still like to see this component be renamed as it has nothing json-specific.